lib/src/sql/value/value.rs

```diff
@@ -464,6 +464,77 @@ impl From<Id> for Value {
 	}
 }
 
+impl TryFrom<Value> for i64 {
+	type Error = Error;
+	fn try_from(value: Value) -> Result<Self, Self::Error> {
+		match value {
+			Value::Number(x) => x.try_into(),
+			_ => Err(Error::TryFromError("Value", "i64")),
```

Comment:
Might it be better to have the actual Value that is attempting to be converted as opposed to just 'Value' (in all of these errors)?

Maybe we should...

```rs
v => Err(Error::TryFromError(value.to_string(), "i64")),
```

Reply:
What do you thing @lunchspider ?

Reply:
This seems great!
I will implement it